src/main/resources/jira/CheckIssue.py

```diff
@@ -31,8 +31,9 @@
         data = json.loads(response.getResponse())['fields']['status']
         issueStatus = data['name']
         print "\nIssue %s status is %s." % (issueId, issueStatus)
-        if issueStatus == expectedStatus:
-            print "\nThe status of issue %s is 'Resolved'" % (issueId)
+        if (issueStatus in expectedStatusList):
+            print "\nThe status of issue %s is %s" % (issueId, issueStatus)
+            finalStatus = issueStatus
             break
         else:
             print "\nIssue is not correct status"
```

Comment:
Should we change this to some plural case?  "Issue is not set to one of the desired statuses"

Reply:
I think it's okay